documentation/plain_rules.md

```diff
@@ -343,4 +343,138 @@ double underscore prefix, so an attribute `x` with fields `a`, `b`, `c`, will al
+                   "value":"40.418889, -3.691944",
+                   "metadatas":[
+                      {
+                         "name":"location",
+                         "type":"string",
+                         "value":"WGS84"
+                      }
+                   ]
+                }
+             ]
+          }
+       }
+    ]
+ }
+ ```
+ 
+will propagate to the core, (and so making available to the EPL sentence) the fields `position__lat`, `position__lon` ,
+`position__x`, `position__y`
+
+```json
+{  
+   "noticeId":"169b0920-8edb-11e6-838d-0b633312661c",
+   "id":"Car1",
+   "type":"Vehicle",
+   "isPattern":"false",
+   "subservice":"/",
+   "service":"unknownt",
+   "position":"40.418889, -3.691944",
+   "position__type":"coords",
+   "position__metadata__location":"WGS84",
+   "position__metadata__location__type":"string",
+   "position__lat":40.418889,
+   "position__lon":-3.691944,
+   "position__x":657577.4234800448,
+   "position__y":9591797.935076647
+}
+```
+
+Analogously, a notification in "geopoint" format, like
+
+```json
+{
+   "subscriptionId":"57f73930e0e2c975a712b8fd",
+   "originator":"localhost",
+   "contextResponses":[
+      {
+         "contextElement":{
+            "type":"Vehicle",
+            "isPattern":"false",
+            "id":"Car1",
+            "attributes":[
+               {
+                  "name":"position",
+                  "type":"geo:point",
+                  "value":"40.418889, -3.691944"
+               }
+            ]
+         },
+         "statusCode":{
+            "code":"200",
+            "reasonPhrase":"OK"
+         }
+      }
+   ]
+}
+```
+
+will send to core an event with the fields `position__lat`, `position__lon`, `position__x`, `position__y` also
+
+```json
+{  
+   "noticeId":"7b8f1c50-8eda-11e6-838d-0b633312661c",
+   "id":"Car1",
+   "type":"Vehicle",
+   "isPattern":"false",
+   "subservice":"/",
+   "service":"unknownt",
+   "position":"40.418889, -3.691944",
+   "position__type":"geo:point",
+   "position__lat":40.418889,
+   "position__lon":-3.691944,
+   "position__x":657577.4234800448,
+   "position__y":9591797.935076647
+```
+
+An example of rule taking advantage of these derived attributes could be:
+
+```json
+{
+    "name": "rule_distance",
+    "text": "select *, \"rule_distance\" as ruleName from pattern [every ev=iotEvent(Math.pow((cast(cast(position__x?,String),float) - 618618.8286057833), 2) + Math.pow((cast(cast(position__y?,String),float) - 9764160.736945232), 2) < Math.pow(500e3,2))]",
+    "action": {
+        "type": "email",
+        "template": "${id} (${type}) is at ${position__lat}, ${position__lon} (${position__x}, ${position__y})",
+        "parameters": {
+            "to": "someone@tid.es",
+            "from": "system@iot.tid.es",
+            "subject": "${id} is coming"
+        }
+    }
+}
+```
+
+that will send an email when the entity with attribute `position` is less than 500 km far away from Cuenca.
```

Comment:
Good example. However, it could be a bit cryptic without the proper background, so some hints about how it works may help. In particular, to mention that we are taking advantage of the circle inequality ```(x-x0)^2 + (y-y0)^2 < r^2``` being (x0, y0) the Cuenca city location. In addition, probably we should mention that we are assuming a simplification, considering the Earth surface as a plane, which may interoduce precission errors.